### PR TITLE
Plotting: added processors

### PR DESCRIPTION
**Pull request by martinfalk**
additional functionality for handling DataFrames
* export to CSV
* convert Image and Volume to DataFrame

### CODE REVIEW

File: modules/plotting/datastructures/dataframeutil.cpp [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *********************************************************************************/
+
+#include <inviwo/core/datastructures/buffer/buffer.h>
+#include <modules/dataplotting/datastructures/dataframeutil.h>
```
Comment (r-englund) [inline modules/plotting/datastructures/dataframeutil.cpp]: Copied from ressearch-dev?  We dont have a dataplotting module

Comment (martinfalk) [reply modules/plotting/datastructures/dataframeutil.cpp]: my bad...

**Review by weinkauf — APPROVED** [review]
Looks good to me!

**Comment by JochenJankowai** [comment]
`/*
            It's deadline time so I will just assume that the volume has only one channel because
            for the data I am using now that is true
            */`

Maybe we should remove my comments before...

File: modules/plotting/processors/volumetodataframe.cpp [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    const auto omitOutliers = omitOutliers_.get();
+    const auto threshold = threshold_.get();
+
+    if (omitOutliers) {
+        for (auto vol : volumeSequence) {
+            if (vol->getDataFormat()->getNumericType() != NumericType::Float) continue;
+            if (vol->getDataFormat()->getComponents() != 1)
+                LogWarn("This volume is omitted because it has more than one channel.");
+            /*
+            It's deadline time so I will just assume that the volume has only one channel because
```
Comment (JochenJankowai) [inline modules/plotting/processors/volumetodataframe.cpp]: Maybe we should remove my comments before...

**Comment by martinfalk** [comment]
superseded by cleaned-up PR #21

**Pull request closed without merging**